content/_index.md

```diff
@@ -1,9 +1,27 @@
 ---
-title: My Site
+title: AI Covenant
 toc: false
 ---
 
-This is the landing page.
+The use of "Artificial Intelligence" models affects the dynamics between people. We encourage mindful and considerate use of "AI" in our community that is grounded in our community values.
+
+Specific behaviors we do not allow:
+
+- Passing generated content off as your own
+- Sending your AI assistant to meetings in your place
+- [promoting the use of AI for everything (AI-ifying everything)]
```

Comment:
This point is kind of covered below when we said "Justify the use of "AI" on a case by case basis, taking into account risks, harms."